AGGREGATE.md

```diff
@@ -255,8 +255,8 @@ Note: if [debugging](https://github.com/WICG/conversion-measurement-api/blob/mai
 is enabled, additional debug fields will be present in aggregatable reports,
 including the cleartext payloads, to allow downstream systems to verify that
 reports are constructed correctly. If debugging is enabled, the `shared_info` 
-will include the flag `debug_mode: 1` to allow the aggregation service to 
-support debugging functionality on debug reports.
+will include the flag `"debug_mode": "enabled"` to allow the aggregation service
```

Comment:
If we're saving characters, "on" is shorter than "enabled" :)
LGTM as-is though.